src/components/Registration/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React , { useEffect, useState }from "react";
+import { Form } from 'react-bootstrap';
+import { useSelector } from "react-redux";
+import { useHistory } from 'react-router-dom';
+import useAuthHooks from '../../redux/Auth/auth-actions';
+import { LoginInner , LoginLogo} from '../login/login.style'
+import {ButtonStyled as Button} from './registration.style'
+import {MainCart} from '../Main/main.style'
+import {LoginForm} from '../login/login.style'
+
+const Register = () => {
+
+    const [name, setName] = useState(" ");
+    const [email, setEmail] = useState(" ");
+    const [password, setPassword] = useState("");
+    const { signupAction } = useAuthHooks();
```

Comment:
Avoid maintaining so many states, try using form validation package
That's convinent and efficient